client/shaders/volumetric_light/opengl_fragment.glsl

```diff
@@ -46,7 +46,7 @@ float sampleVolumetricLight(vec2 uv, vec3 lightVec, float rawDepth)
 		if (min(samplepos.x, samplepos.y) > 0. && max(samplepos.x, samplepos.y) < 1.)
 			result += texture2D(depthmap, samplepos).r < 1. ? 0.0 : 1.0;
 	}
-	return result / samples;
+	return result / samples * pow(texture2D(depthmap, uv).r, 128.0);
```

Comment:
Can you explain this one?
Maybe add a short comment here in the code, too.

Reply:
So basically, the closer an object is to us, the less air we are looking through that scatters light, hence the pixel should not be as bright. The exponent of 128 is just a value that made it look reasonable to me.